config/a200/slam.yaml

```diff
@@ -12,8 +12,9 @@ slam_toolbox:
     # ROS Parameters
     odom_frame: odom
     map_frame: map
+    map_name: /map
```

Comment:
Should we add a note not to change these here and only in the `launch/slam.launch.py`?

Reply:
Done in all the files